application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -85,43 +90,42 @@ private static RestAction<ReminderRoute> computeReminderRoute(JDA jda, long chan
     }
 
     private static RestAction<ReminderRoute> createGuildReminderRoute(JDA jda, long authorId,
-            MessageChannel channel) {
+                                                                      MessageChannel channel) {
         return jda.retrieveUserById(authorId)
-            .onErrorMap(error -> null)
-            .map(author -> ReminderRoute.toPublic(channel, author));
+                .onErrorMap(error -> null)
+                .map(author -> ReminderRoute.toPublic(channel, author));
     }
 
     private static RestAction<ReminderRoute> createDmReminderRoute(JDA jda, long authorId) {
         return jda.openPrivateChannelById(authorId).map(ReminderRoute::toPrivate);
     }
 
-    private static void sendReminderViaRoute(RestAction<ReminderRoute> routeAction, long id,
-            CharSequence content, TemporalAccessor createdAt) {
+    private void sendReminderViaRoute(RestAction<ReminderRoute> routeAction, int id,
+                                      long authorId, long channelId,
+                                      CharSequence content, TemporalAccessor createdAt, long guildId, int reminderFailureCount) {
         Function<ReminderRoute, MessageCreateAction> sendMessage = route -> route.channel
-            .sendMessageEmbeds(createReminderEmbed(content, createdAt, route.target()))
-            .setContent(route.description());
+                .sendMessageEmbeds(createReminderEmbed(content, createdAt, route.target()))
+                .setContent(route.description());
 
-        Consumer<Throwable> logFailure = failure -> logger.warn(
-                """
-                        Failed to send a reminder (id '{}'), skipping it. This can be due to a network issue, \
-                        but also happen if the bot disconnected from the target guild and the \
-                        user has disabled DMs or has been deleted.""",
-                id);
+        Consumer<Throwable> logFailure = failure -> {
+            //
+            reminderFailureAction(id, channelId, authorId, content, guildId, reminderFailureCount);
+        };
```

Comment:
the name `logFailure` is kinda incorrect now. that said, i would just put the lambda without defining it in advance now. so `.queue(doNothing(), failure -> reminderFailureAction(...))`

Reply:
resolved this as well